### DISCUSSION TITLE
Create Z2M Windows 10 Service

### DISCUSSION THREAD

**Discussion by EddieDSuza** [Koenkk/zigbee2mqtt#14149] [General]
Followed to the teeth to setup Z2M on a [windows 10 platform](https://www.zigbee2mqtt.io/guide/installation/05_windows.html)

However the guide stops shot on setting up the service that auto starts on a reboot. 

Assistance would be appreciated.

**Comment by hsdneo** [comment]
Same question here. Today I installed Z2M on my windows machine where my MQTT Broker and OH is running and it would be great that this is running automatically in the background.

**Reply by hsdneo** [reply]
Any help/hint on this?

**Reply by hsdneo** [reply]
I found a suitable solution for me: https://forums.homeseer.com/forum/hs4-products/hs4-plugins/lighting-primary-technology-plug-ins-aa/zigbee-plus-homeseer/1620739-auto-start-restart-and-monitoring-of-zigbee2mqtt
